### ISSUE TITLE
Update url_launcher package

### ISSUE THREAD

**Issue by MingSern** [timtraversy/new_version#4]
Currently my app is using url_launcher package with the latest version, when trying to install this new_version package, this error message pops up:

`$ flutter pub get`
"Because every version of new_version depends on url_launcher ^4.0.3 and <MY_APP_NAME> depends on url_launcher ^5.4.1, new_version is forbidden. So, because <MY_APP_NAME> depends on new_version ^0.0.4, version solving failed."

I believe you need to update your url_launcher package to solve this problem, thank you.

**Comment by timtraversy** [comment]
Thanks! Created a branch with this small change. Would you mind testing it to see if it works? I don't do much flutter these days to be honest and don't have the env set up.

You can see how here: https://dart.dev/tools/pub/dependencies#path-packages

So clone this repo, check out the `url_launcher_version_increment` branch, and add to your pubspec:

```
dependencies:
  new_version:
    path: /path/on/your/computer/to/new_version
```

Let me know.

**Comment by aalieco** [comment]
First of all, thanks for the great effort. I could help you out a bit, since I will be doing the same fix on my local machine anyway.

Secondly, you forgot to add return versionStatus; at the end of _getAndroidStoreVersion which made the lib not useable for Android. I can push that fix for you.

Finally, I want to improve the comparison logic, to check for the version number and not just check for whether the strings are equal or not.

**Comment by timtraversy** [comment]
I’d be happy to review any PRs you submit, thanks for offering.

**Comment by timtraversy** [comment]
This is resolved in the most recent release, 0.0.5. Thanks for raising the issue.
